emcc.py

```diff
@@ -2439,8 +2439,6 @@ def phase_linker_setup(options, state, newargs):
     settings.JS_LIBRARIES.append((0, 'library_pthread_stub.js'))
 
   if settings.MEMORY64:
-    if settings.ASYNCIFY == 1 and settings.MEMORY64 == 1:
```

Comment:
Perhaps this can be changed to `if settings.ASYNCIFY == 2 and settings.MEMORY64 == 2:`  with the error `MEMORY64=2 is not compatible with JSPI (ASYNCIFY=2)`?